examples/pom.xml

```diff
@@ -23,7 +23,7 @@
     <dependency>
       <groupId>commons-cli</groupId>
       <artifactId>commons-cli</artifactId>
-      <version>1.2</version>
+      <version>1.3.1</version>
```

Comment:
AFAIK, it is fully backwards compatible. I was using 1.3 syntax style in the original [tachyon-fuse](https://github.com/ibm-research-ireland/tachyon-fuse) project, and that's why I ported the dependency here.